packages/design-system-react/src/components/Box/Box.types.ts

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+export enum BoxBackgroundColor {
+  BackgroundDefault = 'bg-background-default',
+  BackgroundAlternative = 'bg-background-alternative',
+  PrimaryDefault = 'bg-primary-default',
+  PrimaryMuted = 'bg-primary-muted',
+  ErrorDefault = 'bg-error-default',
+  ErrorMuted = 'bg-error-muted',
+  SuccessDefault = 'bg-success-default',
+  SuccessMuted = 'bg-success-muted',
+  InfoDefault = 'bg-info-default',
+  InfoMuted = 'bg-info-muted',
+  WarningDefault = 'bg-warning-default',
+  WarningMuted = 'bg-warning-muted',
+  Transparent = 'bg-transparent',
+}
+
+export enum BoxBorderColor {
+  BorderDefault = 'border border-border-default',
+  BorderMuted = 'border border-border-muted',
+  PrimaryDefault = 'border border-primary-default',
+  ErrorDefault = 'border border-error-default',
+  SuccessDefault = 'border border-success-default',
+  InfoDefault = 'border border-info-default',
+  WarningDefault = 'border border-warning-default',
+  None = '',
+}
```

Comment:
This is the sort of thing that I like. Defining a set of common, human readable configurations that we can use our Typescript compiler to suggest 👍